lib/stdlib.cf

```diff
@@ -71,6 +71,9 @@ bundle common stdlib_common
       "input[cfe_internal_hub]"           string => "$(this.promise_dirname)/cfe_internal_hub.cf";
       "input[cfengine_enterprise_hub_ha]" string => "$(this.promise_dirname)/cfengine_enterprise_hub_ha.cf";
 
+      # jUnit reports
+      "input[junit]"                      string => "$(this.promise_dirname)/junit.cf";
```

Comment:
Yeah agent bundles get pre-eval for vars. But they should all be quickly skipped since the vars wont have values unless its passed as a parameter.